### ISSUE TITLE
Mssql table default column value is a function produced error

### ISSUE THREAD

**Issue by qiangxue** [yiisoft/yii#272]
I know I get this error for SQL server, but I don't know if it applies to the other databases.

<b>What steps will reproduce the problem?</b>
1. Create a table in SQL server like this:
CREATE TABLE [dbo].[Test](
   [id] [int] IDENTITY(1,1) PRIMARY KEY CLUSTERED,
   [data] [varchar](50) COLLATE SQL_Latin1_general_CP1_CI_AS NULL,
   [dateAdded] [datetime] NOT NULL CONSTRAINT [DF_Test_dateAdded] DEFAULT (getDate()),
) ON [PRIMARY]

2. Generate a model class Test in gii

3. Create a new Test object and save it like this:
 $t = new Test();
 $t-&gt;data = 'Hi!';
 $t-&gt;save();

<b>What is the expected output? What do you see instead?</b>
I would expect no output, but instead I get an error which says:
CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 10007 Conversion failed when converting datetime from character string. [10007](severity 5)[(null)]. The SQL statement executed was: INSERT INTO [dbo].[Test]([dateAdded],[data]) VALUES(:yp0, :yp1)

<b>What version of the product are you using? On what operating system?</b>
1.1.8.r3324 on Windows NT.

<b>Please provide any additional information below.</b>
Basically, the ActiveRecord reads the default value of dateAdded as the string, 'getdate' and then tries to set it when saving, which throws an error. Is there some way to change activerecord to not set defaults for columns with functions as defaults? or to set ignore columns for insert/update? or to somehow change the function into the PHP equivalent?<p>Migrated from http://code.google.com/p/yii/issues/detail?id=3122</p>
